.github/workflows/label-jetbrains-issues.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto-label Issues
+
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/github-script@v7
+        with:
+          script: |
+            const body = context.payload.issue.body || '';
+            const labels = context.payload.issue.labels.map(l => l.name);
+            
+            // Check if JetBrains Plugin is selected
```

Comment:
Do we want to add here additional labels as well like 'VS Code'? since we are expanding cline it might be worth to have labels for things like the CLI